helix-rest/src/main/java/org/apache/helix/rest/clusterMaintenanceService/MaintenanceManagementService.java

```diff
@@ -144,6 +148,26 @@ public MaintenanceManagementService(ZKHelixDataAccessor dataAccessor,
     _namespace = namespace;
   }
 
+  @VisibleForTesting
+  MaintenanceManagementService(MaintenanceManagementServiceBuilder builder) {
```

Comment:
This is not right way of using builder. Use builder we don't want them to create instance from constructor and it should be private constructor.

Reply:
Fixed